lua/lvim/plugins.lua

```diff
@@ -159,10 +165,10 @@ local core_plugins = {
   -- Comments
   {
     "numToStr/Comment.nvim",
```

Comment:
Load order might be a problem here
When doing the following
Open lvim > find file using Telescope > hit the comment line keybind <leader>/

Which-key will need to list comment.nvim as a dependency for <leader>/ to work.

gcc,gcb work well though.